src/ScriptingCore/utf8_tools.cpp

```diff
@@ -17,24 +17,39 @@ Copyright 2009 Dan Weatherford, Facebook inc
 
 #include <limits.h>
 #include <boost/algorithm/string/case_conv.hpp>
-#include <locale>
-#include <codecvt>
+#include <boost/locale/encoding_utf.hpp>
+#include <string>
+
+///#include <codecvt>
+//#include <locale>
 
 #include "utf8_tools.h"
 
-std::wstring_convert<std::codecvt_utf8<wchar_t>> utf8_conv;
```

Comment:
I got this compilation error
```bash
Building CXX object firebreath/ScriptingCore/CMakeFiles/ScriptingCore.dir/utf8_tools.cpp.o
cd /home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/build/firebreath/ScriptingCore && /usr/lib64/ccache/c++    -DFB_X11=1 -DXP_UNIX -fPIC -DBOOST_ALL_NO_LIB=1 -DUNICODE -D_UNICODE -DFB_X11=1 -DXP_UNIX -fPIC -DBOOST_ALL_NO_LIB=1 -DUNICODE -D_UNICODE -I/home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/ScriptingCore -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/freetype2 -I/home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/config -I/home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/PluginCore -I/home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/3rdParty/boost    -std=gnu++11 -o CMakeFiles/ScriptingCore.dir/utf8_tools.cpp.o -c /home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/ScriptingCore/utf8_tools.cpp
/home/nikita/work/Projects/rucrypto/rtPluginPINPad/external/firebreath/src/ScriptingCore/utf8_tools.cpp:21:19: fatal error: codecvt: No such file or directory
 #include <codecvt>
```
now I think there should be a better solution...